src/foam/nanos/notification/email/EmailTemplateApplyEmailPropertyService.js

```diff
@@ -39,9 +39,9 @@ foam.CLASS({
       String locale = (String) templateArgs.get("locale");
 
       // STEP 1) Find EmailTemplate
-      EmailTemplate emailTemplateObj = DAOResourceLoader.findTemplate(x, templateName, group, locale);
-      if ( emailTemplateObj == null ) {
-        logger.error(this.getClass().getSimpleName(), "EmailTemplate not found", templateName, group);
+      EmailTemplate emailTemplateObj = DAOResourceLoader.findTemplate(x, templateName, group, locale, emailMessage.getSpid(), templateArgs);
+      if ( emailTemplateObj == null || ! emailTemplateObj.getEnabled() ) {
+        logger.error(this.getClass().getSimpleName(), "EmailTemplate not found or disabled", templateName, group);
```

Comment:
! enabled is not an error. 
if emailTemplateObj is null, report that, but report debug for not enabled.

Reply:
AGREE with Joel
thinking of just wrapping the try in // STEP 2) Apply Template to emailMessage
and not logging anything.
Who cares if it is not enabled.

Reply:
```
if ( emailTemplateObj == null ) {
        logger.error(this.getClass().getSimpleName(), "EmailTemplate not found", templateName, group);
}
If ( ! emailTemplateObj.getEnabled() ) {
        logger.debug(this.getClass().getSimpleName(), "EmailTemplate disabled", templateName, group);
}
return emailMessage;
```
Brings up a point, that we never ttrack something that doesn't have an email ... which is good for now, we'll just have to check how many times these loggers are in the jrl.